samples/sampler/stories/qa/Text.js

```diff
@@ -17,6 +17,10 @@ import Repeater from '@enact/ui/Repeater';
 
 import Section from './components/KitchenSinkSection';
 
+const Config = {
+	displayName: 'Font-weight'
```

Comment:
Can you rename this to "Text" ? It will allow further knobs about the text to be added